.pre-commit-config.yaml

```diff
@@ -94,13 +94,19 @@ repos:
     files: ^tools/.*$
     types: [text, python]
 - repo: https://github.com/shellcheck-py/shellcheck-py
-  rev: v0.8.0.3
+  rev: v0.8.0.4
   hooks:
   - id: shellcheck
 - repo: https://github.com/DavidAnson/markdownlint-cli2
   rev: v0.4.0
   hooks:
   - id: markdownlint-cli2
+- repo: https://github.com/sirosen/check-jsonschema
+  rev: 0.10.2
+  hooks:
+  - id: check-github-actions
+  # TODO: Checking .github/workflows/build.yml fails
+  #- id: check-github-workflows
```

Comment:
I don't really understand the purpose of this so far. We don't have any github actions and and the workflows are not being checked.